src/main/java/org/folio/circulation/resources/handlers/error/CirculationErrorType.java

```diff
@@ -24,6 +24,7 @@
 
   // Errors related to the loan
   FAILED_TO_FIND_SINGLE_OPEN_LOAN,
+  AT_LEAST_ONE_ITEM_HAS_OPEN_LOAN,
```

Comment:
Move to // Errors related to requests
ONE_OF_INSATANCES_ITEMS_HAS_OPEN_LOAN

Reply:
done